components/footer.js

```diff
@@ -0,0 +1,40 @@
+import Link from "next/link";
+
+export default function Footer() {
+  return (
+    <footer className="text-white bg-blue-500 font-big py-3 px-8 md:py-8">
+      <nav className="flex justify-between item-center mx-auto"> 
+        <ul className="flex item-center mr-3 text-2xl md:text-3xl">
+          <li>
+            <Link href="/">
+              <a title="OpenTech">OpenTech</a>
+            </Link>
+          </li>
+        </ul>
+        <ul className="flex items-center overflow-auto text-xs">
+          <li className="text-center w-40 mr-6 md:m-0">
+            <Link href="/https://github.com/opentech-pe/opentech.pe/issues/new?assignees=jhonfitzgerald%2C+paulotijero&labels=cfp&template=propuesta-de-charla.md&title=Título+de+tu+charla+.+.+." target="_blank" rel="noopener noreferrer">
+              <a title="Dar una charla">
+                Dar una charla
+              </a>
+            </Link>
```

Comment:
In this block is not necessary to use the `Link` component because redirect URL to another page, please in this cases only use label `a`
```suggestion
            <a title="Dar una charla" href="https://github.com/opentech-pe/opentech.pe/issues/new?assignees=jhonfitzgerald%2C+paulotijero%2C+JimmyLoloy98&labels=cfp&template=propuesta-de-charla.md&title=T%C3%ADtulo+de+tu+charla+.+.+." target="_blank" rel="noopener noreferrer">
              Dar una charla
            </a>
```